addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -58,6 +55,17 @@ def _l10n_it_edi_generate_electronic_invoice_filename(self, invoice):
             'progressive_number': progressive_number.zfill(5),
         }
 
+    def _l10n_it_edi_get_message(self, code):
```

Comment:
Some sentences are called multiple times, I'll put them here, instead of having different copies.